Code/Framework/AzToolsFramework/AzToolsFramework/AssetEditor/AssetEditorBus.h

```diff
@@ -57,8 +57,9 @@ namespace AzToolsFramework
         public:
             static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Single;
 
-            //! Open Asset Editor (if it is not already open) and create a new asset of type specified, if possible
-            virtual void CreateNewAsset(const AZ::Data::AssetType& assetType) = 0;
+            //! Open Asset Editor (if it is not already open) and create a new asset of type specified, if possible.
+            //! Inform the specified component when complete.
+            virtual void CreateNewAsset(const AZ::Data::AssetType& assetType, const AZ::Uuid& interestedComponentId) = 0;
 
```

Comment:
Same comments about this may not be coming from a component so a more generic name is preferable

Reply:
Changed